test/functional-tests/test.sh

```diff
@@ -43,23 +45,22 @@ function run_script {
     echo "run script"
     echo "###################################"
     killall dunst
-    PATH=".:$PATH" ../../dunst -config dunstrc.run_script &
-    ../../dunstify -a "dunst tester" -u c \
+    PATH=".:$PATH" $DUNST -config dunstrc.run_script &
```

Comment:
Why didn't you use `start_dunst` here?

Reply:
Because PATH was exported. I don't know if it would work with the function. I will try

Reply:
I refactored a bit so this can be handled nicely